content/dart/concepts/map/terms/remove/remove.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.remove()'
+Description: 'Removes the first occurence of a specified value from the list.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Methods'
+  - 'Dart'
+  - 'Lists'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, the **`.remove()`** method is used to remove the first occurence of a specified value from the list. This method is useful when a particular element from a list has to be removed without knowing it's index. It returns true if the specified value is removed from the list.
```

Comment:
This entry is for Map concept so you need to write the content in reference to Maps. You have mentioned lists everywhere in the content, please replace it.